tests/functional/monitoring/conftest.py

```diff
@@ -1133,3 +1133,76 @@ def teardown():
     teardown()
 
     return measured_op
+
+
+@pytest.fixture
+def measure_change_client_ocs_version_and_stop_heartbeat(
+    request, measurement_dir, threading_lock
+):
+    """
+    Change ocs version of client to a different number, measure the time when it was
+    rewritten and alerts that were triggered during this event. To achieve the change
+    will be also stopped heartbeat cron job on the client to ensure that the version
+    is not rewritten.
+
+    Returns:
+        dict: Contains information about `start` and `stop` time for rewritting
+            the client version
+
+    """
+
+    current_version = storageconsumer.get_ocs_version()
+    logger.info(f"Reported client version: {current_version}")
+    original_cluster = config.cluster_ctx
+    logger.info(f"Provider cluster key: {original_cluster}")
+    logger.info("Switch to client cluster")
+    config.switch_to_consumer()
+    client_cluster = config.cluster_ctx
+    logger.info(f"Client cluster key: {client_cluster}")
+    cluster_id = run_cmd(
+        "oc get clusterversion version -o jsonpath='{.spec.clusterID}'"
+    )
+    client_name = f"storageconsumer-{cluster_id}"
+    client = storageconsumer.StorageConsumer()
+
+    def change_client_version():
+        """
+        Stop heartbeat and change value of ocs version in storage client resource
+        for 3 minutes.
+
+        """
+        nonlocal client
+        nonlocal original_cluster
+        # run_time of operation
+        run_time = 60 * 3
+        client.stop_heartbeat()
+        client.change_ocs_version()
```

Comment:
can't find **change_ocs_version** method. Maybe missed something, but class StorageConsumer is a newly added class and the closest it has is **set_ocs_version**